tree/master/cloud/src/solution/Deployment/Scripts/Deploy-SPOFiles.ps1

```diff
@@ -11,6 +11,10 @@ Add-Type -TypeDefinition @"
 # Function to display message on console
 Function Show-Message([string] $Message, [string] $Type, [bool] $Newline = $true)
 {
+
+	# Set log file path
+$LogFile = "$ScriptDirectory\Logs\Log.txt"
```

Comment:
$logfile should be set outside of this function as it user across multiple functions.  if required move it to before function declaration from bottom of the file.